holdings_maintenance/pds4/pds4infoshelf.py

```
@@ -0,0 +1,896 @@
+
+                (bytes1, count1, modtime1, checksum1, size1) = dirinfo
+                (bytes2, count2, modtime2, checksum2, size2) = shelfinfo
+
+                # Truncate modtimes to seconds
+                modtime1 = modtime1.rpartition('.')[0]
+                modtime2 = modtime2.rpartition('.')[0]
+
+                agreement = True
+                if bytes1 != bytes2:
+                    logger.error('File size mismatch %d %d' %
+                                    (bytes1, bytes2), key)
```

Flake8 (7.2.0)

[error] continuation line over-indented for visual indent

(E127)